src/candy-machine/candy-machine.service.ts

```diff
@@ -163,18 +197,17 @@ export class CandyMachineService {
       where: { comicIssueId: comicIssue.id },
     });
 
-    const candyMachineKey = Keypair.generate();
-
     let darkblockId = '';
     if (collectionNft) {
       collectionNftAddress = new PublicKey(collectionNft.address);
+      // TODO: Either store darkblockId in collection or fetch the id from offchain metadata
     } else {
       let darkblockMetadataFile: MetdataFile;
       if (comicIssue.pdf) {
         darkblockId = await this.darkblockService.mintDarkblock(
```

Comment:
I'm seeing a lot of these in the code, my thought is we should start introducing name parameters instead of positional ones, thoughts?
Like instead of this we would have something like:
```
{ 
pdf: comicIssue.pdf,
description: comicIssue.description,
creatorAddress: comicIssue.creatorAddress
};
```
I'm not saying we should go over the code and clean this up, I'm just make an suggestion and we can make a changes when it's incidental to us.